alica_tests/include/alica_tests/plans/TestStandardLibraryCompareConditions.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include <alica_tests/util/AlicaTestsPlan.h>
+#include <boost/dll/alias.hpp>
+#include <engine/BasicUtilityFunction.h>
+#include <engine/DefaultUtilityFunction.h>
+#include <engine/UtilityFunction.h>
+
+namespace alica
+{
+class TestStandardLibraryCompareConditions : public AlicaTestsPlan<TestStandardLibraryCompareConditions>
```

Comment:
Added tests for all types to test EqualTo and LessThan conditions. Not adding tests for other condition as internally I am using same functions, so it would be redundant.